### PR TITLE
OPENTOK-47695: Extend OAuth TTL to 3 hours

### PR DESCRIPTION
**Pull request by v-kpheng**
#### What is this PR doing?
Increases TTL of OAuth from 30 minutes to 3 hours

#### How should this be manually tested?
N/A

#### What are the relevant tickets?
https://jira.vonage.com/browse/OPENTOK-47695

### CODE REVIEW

**Review by albertoacn — DISMISSED** [review]
LGTM

File: server/roomstore.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 const OpenTok = require('opentok');
 
-const ROOM_INACTIVITY_TIME_LIMIT = 30 * 60 * 1000; // 30 minutes; minutes x seconds x milliseconds
+const NUM_MILLISECONDS_PER_HOUR = 60 * 60 * 1000;
+const ROOM_INACTIVITY_TIME_LIMIT = 3 * NUM_MILLISECONDS_PER_HOUR;
```
Comment (maikthomas) [inline server/roomstore.js:4]: I feel like `ONE_HOUR_IN_MS` in clearer (and something I've seen before in other projects

**Review by maikthomas — CHANGES_REQUESTED** [review]
Can we change this to read an env var instead? and leave a default

**Review by maikthomas — APPROVED** [review]
LGTM

**Pull request merged**